agimus_demo_03_mpc_dummy_traj/config/mpc_params.yaml

```diff
@@ -1,32 +1,64 @@
-agimus_controller_node:
-  ros__parameters:
-    ocp:
-      dt: 0.01
-      horizon_size: 20
-      armature: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
-      gripper_weight: 500
-      state_weight: 5
-      control_weight: 0.001
-      max_qp_iter: 100
-      max_iter: 10
-      use_constraints: True
-      activate_callback: False
-      increasing_weights:
-        max: 100.0
-        percent: 0.9
-        time_reach_percent: 6
-      effector_frame_name: "fer_link8"
-    moving_joint_names:
-      [
-        "fer_joint1",
-        "fer_joint2",
-        "fer_joint3",
-        "fer_joint4",
-        "fer_joint5",
-        "fer_joint6",
-        "fer_joint7",
-      ]
-    save_predictions_and_refs: True
-    rate: 100
-    package_name: "agimus_demo_03_mpc_dummy_traj"
-    path_to_yaml: "config/mpc_params.yaml"
+#agimus_controller_node:
+#  ros__parameters:
+ocp:
```

Comment:
This file has to be in the `agimus_coontroller_ros` package it's part of the node API. This is the decalaration of the Paramters.
While the lines you deleted can be considered as 1 instance of these parameters. Hence they live in the demo package.